.gitignore

```diff
@@ -9,3 +9,7 @@ node_modules
 pmm-app.tar.gz
 pmm-app/dist/
 pmm-app/coverage/
+pmm-app/video/
+pmm-app/pr.browsers.json
+pmm-app/tests/output/
+pmm-app/jest.config.js
```

Comment:
Do we mean here that everyone is supposed to have his/her own jest config? Then we'd need a sample.

Reply:
It's autogenerated config added by grafana toolkit